ext/droption/droption.h

```diff
@@ -453,6 +453,17 @@ droption_t<unsigned int>::convert_from_string(const std::string s)
     return true;
 }
 template<> inline bool
+droption_t<double>::convert_from_string(const std::string s) {
+  double input = atof(s.c_str());
+  if (input >= 0)
```

Comment:
Why would the result need to be non-negative?  Maybe strtod should be used instead of atof to detect errors.

Reply:
Changed to strtod.